CHANGELOG.md

```diff
@@ -6,7 +6,9 @@
 
 ### What's new
 
-- **Added number range filter to `Table`.** Use `tableFilters.NumberRangeFilter`.
+- **Added number range filter to `Table`.** Use `tableFilters.NumberRangeFilter`.\*\*
+- **Update `ThemeProvider` component to allow optional owningDocument prop** to allow use in popup browser windows.
+- **Update `useTheme` hook to allow specification of owningDocument** to allow use in popup browser windows.
```

Comment:
These two can be combined.

Reply:
Also they should use past tense to be consistent.

```suggestion
- **Added `ownerDocument` prop to `ThemeProvider` and `useTheme`** to allow use in popup browser windows.

Reply:
updated